src/lua/log.lua

```diff
@@ -389,6 +410,16 @@ local function log_configure(self, cfg)
         log_cfg[o] = cfg[o]
```

Comment:
Sorry, my mistake. Looks like current and given `level_modules` are merged. It just we don't test it. Let's add test also.

Still I think it will be nice to have resetting individual log levels using `box.NULL` as described above.

Reply:
Added the test and reset of individual log levels using `box.NULL`.